brotli_asgi/__init__.py

```diff
@@ -147,10 +147,14 @@ async def send_with_brotli(self, message: Message) -> None:
                 # Don't apply Brotli to small outgoing responses.
                 await self.send(self.initial_message)
                 await self.send(message)
+            headers = MutableHeaders(raw=self.initial_message["headers"])
+            if "br" in headers.get("Content-Encoding", ""):
```

Comment:
@rhysrevans3 I'm wondering: is OK here to check if there is the `br` compression explicitly, or should the middleware to prevent compressing again if there is_any_ other compression? (`"Content-Encoding" in headers`)?